misc: improve bug report issue template (#4878)
facebook/docusaurus: .github/ISSUE_TEMPLATE/bug.md

## 🐛 Bug Report

### Prerequisites

<!--

> IMPORTANT: Please meet all following the preconditions before submitting an issue.
> Put an X between the brackets on each line to confirm you have completed them.

Reviewer: ```suggestion
IMPORTANT: Please check the following items before creating a issue.
Mark the checkbox by putting an X between the bracket.
This way we know you've done these steps first.
```

-->

> - [ ] Are you using the latest version of Docusaurus?
> - [ ] Have you tried running the `yarn clear` command?

Reviewer: ```suggestion
- [ ] I'm using the latest version of Docusaurus.
- [ ] I have tried the `yarn clear` command.
```

## 🐛 Bug Report

### Prerequisites

<!--
IMPORTANT: Please check the following items before creating a issue.
Mark the checkbox by putting an X between the bracket.
This way we know you've done these steps first.
-->

- [ ] I'm using the latest version of Docusaurus.

> - [ ] I have tried the `yarn clear` command.

Reviewer 2: ```suggestion
- [ ] I have tried the `npm run clear` or `yarn clear` command.
- [ ] I have tried `rm -rf node_modules yarn.lock package-lock.json` and re-installing packages
```

I think showing the npm command might be useful for non-frontend-dev users that might be confused by seeing a yarn command.

Also think re-generating lockfile / reinstall is a useful thing to try

> - Public site url:

Reviewer: ```suggestion
- Public site URL:
```

While we're fixing things up, URL is an abbreviation, and in English we capitalize abbreviations. 😄
